### PR TITLE
[12.0] [MIG] partner_second_lastname

### PR DESCRIPTION
**Pull request by Nikul-Chaudhary**
Partner second lastname
=======================

This module was written to extend the functionality of ``partner_firstname`` to support having a second lastname for contact partners.

In some countries, it's important to have a second last name for contacts.

Contact partners will need to fill at least one of the name fields (*First name*, *First last name* or *Second last name*).

### CODE REVIEW

**Comment by Nikul-Chaudhary** [comment]
@pedrobaeza Could you please review it?

File: partner_second_lastname/tests/test_onchange.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# © 2015 Grupo ESOC Ingeniería de Servicios, S.L.U.
```
Comment (tarteo) [inline partner_second_lastname/tests/test_onchange.py]: © > Copyright

File: partner_second_lastname/README.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (tarteo) [inline partner_second_lastname/README.rst:1]: Use the split readme.

File: partner_second_lastname/models/res_partner.py [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+from odoo.addons.partner_firstname import exceptions
+
+
+class ResPartner(models.Model):
+    """Adds a second last name."""
+    _inherit = "res.partner"
+
+    lastname2 = fields.Char(
+        "Second last name",
+        oldname="lastname_second"
```
Comment (tarteo) [inline partner_second_lastname/models/res_partner.py]: oldname is not needed

**Review by tarteo — CHANGES_REQUESTED** [review]

**Review by LoisRForgeFlow — COMMENTED** [review]
Please, squash transifex commits.

**Comment by Nikul-Chaudhary** [comment]
@lreficent @tarteo Done

**Review by tarteo — APPROVED** [review]
Thanks.

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**